app/components/modals/tx-sender/withdraw-flow/Summary.tsx

```
@@ -1,50 +1,76 @@
+  onChange,
 }) => (
-  <Container>
-    <Heading size={EHeadingSize.SMALL} level={4} className="mb-4">
+  <section className={styles.contentWrapper}>
+    <Heading
+      className={styles.withSpacing}
+      size={EHeadingSize.HUGE}
+      level={4}
+      decorator={false}
+      disableTransform={true}
+    >
       <FormattedMessage id="withdraw-flow.summary" />
     </Heading>
 
-    <WithdrawTransactionDetails additionalData={additionalData} className="mb-4" />
+    <ButtonArrowLeft
+      className={styles.withSpacing}
+      onClick={onChange}
+      width={ButtonWidth.NO_PADDING}
+    >
+      <FormattedMessage id="modal.sent-eth.change" />
+    </ButtonArrowLeft>
+
+    <WithdrawTransactionDetails additionalData={additionalData} className={styles.withSpacing}>
+      <FormattedMessage id="withdraw-flow.awaiting-confirmation" />
+    </WithdrawTransactionDetails>
 
-    <div className="text-center">
+    <section className="text-center">
       <Button onClick={onAccept} data-test-id="modals.tx-sender.withdraw-flow.summary.accept">
         <FormattedMessage id="withdraw-flow.confirm" />
       </Button>
-    </div>
-  </Container>
+    </section>
+  </section>
 );
 
-export const WithdrawSummary = appConnect<IStateProps, IDispatchProps>({
-  stateToProps: state => ({
-    additionalData: selectTxAdditionalData<ETxSenderType.WITHDRAW>(state)!,
-  }),
-  dispatchToProps: d => ({
-    onAccept: () => d(actions.txSender.txSenderAccept()),
+export const WithdrawSummary = compose<TComponentProps, {}>(
+  appConnect<IStateProps, IDispatchProps>({
+    stateToProps: state => ({
+      additionalData: selectTxAdditionalData<ETxSenderType.WITHDRAW>(state),
+    }),
+    dispatchToProps: d => ({
+      onAccept: () => d(actions.txSender.txSenderAccept()),
+      onChange: () => d(actions.txSender.txSenderChange(ETxSenderType.WITHDRAW)),
+    }),
   }),
-})(WithdrawSummaryComponent);
+  branch<IStateProps>(props => props.additionalData === undefined, renderNothing),
```

hm, i would say we should throw an error here?